### PR TITLE
[Python] Use feather implementation from pyarrow

### PR DESCRIPTION
**Pull request by wesm**

### CODE REVIEW

File: python/setup.py [hunk 1]
```diff
@@ -158,18 +68,12 @@ def run(self):
     packages=['feather', 'feather.tests'],
     version=VERSION,
     package_data={'feather': ['*.pxd', '*.pyx']},
-    ext_modules=extensions,
-    cmdclass={
-        'clean': clean,
-        'build_ext': build_ext
-    },
-    install_requires=['cython >= 0.21'],
+    # install_requires=['cython >= 0.21', 'pyarrow>=0.3.0'],
```
Comment (xhochy) [inline python/setup.py]: Forgot to remove `#`?

Comment (xhochy) [inline python/setup.py]: Also there is probably no requirement on `cython` anymore?

Comment (wesm) [inline python/setup.py]: Good point. will fix

**Comment by wesm** [comment]
I removed Python from Travis CI since there are no longer any Python unit tests here

**Pull request closed without merging**